### ISSUE TITLE
Curio bug? Not sure..

### ISSUE THREAD

**Issue by Rileybutt** [apace100/origins-fabric#167]
I noticed a bug with any origin that has armor restriction (Elytrian if only the default). When you have the Curio mod installed as well, you can completely bypass the armor restriction. I'm not really sure if this is a bug with this mod or Curio itself so I really do apologize if this is wrong. I was going to post this bug on both issue trackers just in case. But yeah, I apologize if this here is incorrect.

It's not too big of an issue either but it can be broken for any servers that are serious about the Elytrians having only light armor.

To recreate, I just have Curios and Origins installed, open Curio menu, place Diamond/Netherite into the armor slot.

**Comment by apace100** [comment]
Yes, this is definitely a bug! Thanks for reporting. I'll look into ways of fixing it.
